lib/gollum/app.rb

```diff
@@ -274,9 +275,13 @@ class App < Sinatra::Base
           return
         end
         committer.commit
-
+        
         # Renaming preserves format, so add the page's format to the renamed path to retrieve the renamed page
-        page = wiki_page("#{rename}.#{Gollum::Page.format_to_ext(page.format)}").page
+        new_path = "#{rename}.#{Gollum::Page.format_to_ext(page.format)}"
+        # Add a redirect from the old page to the new
+        wiki.add_redirect(page.url_path, new_path.gsub(/(^\/)/, '')) if @redirects_enabled
```

Comment:
Alternatively, use the slower but more explicit `clean_url` helper from https://github.com/gollum/gollum/blob/5.x/lib/gollum/helpers.rb#L20

Reply:
Of course, much nicer. Thanks for the suggestion!

Reply:
Done.